contracts/HubPool.sol

```diff
@@ -29,16 +31,14 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
         bool isEnabled;
     }
 
-    enum RefundRequestStatus {
-        Pending,
-        Finalized,
-        Disputed
-    }
-
     struct RelayerRefundRequest {
-        bytes32 poolRebalanceProof;
-        bytes32 destinationDistributionProof;
-        RefundRequestStatus status;
+        uint64 requestExpirationTimestamp;
+        uint64 unclaimedPoolRebalanceLeafs;
```

Comment:
nit: change this name to end with `LeafCounts` instead of `Leafs`. I was a bit confused because without "counts" i expected this to be an array or a bitmap

Reply:
sure.